notebooks/slsim_single_lens_simulation.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
<p>I would make a description for what type of deflector and source model this is valid. You only make the connection later on. Perhaps also add the definition of all these components, or link to the class</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>